src/main/java/net/greatstart/controllers/ProjectController.java

```diff
@@ -1,39 +1,84 @@
 package net.greatstart.controllers;
 
 import net.greatstart.model.Project;
+import net.greatstart.model.User;
 import net.greatstart.services.ProjectService;
+import net.greatstart.services.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
 
+import javax.validation.Valid;
+import java.security.Principal;
 import java.util.List;
 
 @Controller
 public class ProjectController {
+    private static final String REDIRECT_TO_PROJECTS = "redirect:/projects";
 
     private ProjectService projectService;
+    private UserService userService;
 
     @Autowired
-    public ProjectController(ProjectService projectService) {
+    public ProjectController(ProjectService projectService, UserService userService) {
         this.projectService = projectService;
+        this.userService = userService;
     }
 
-    public boolean createProject(Project project) {
-        return projectService.createProject(project);
+    @RequestMapping("/projects")
+    public String showProjects(Model model) {
+        List<Project> projectList = projectService.getAllProjects();
+        model.addAttribute(projectList);
+        return "project/projects";
     }
 
-    public boolean updateProject(long id) {
-        return projectService.updateProject(id);
+    @RequestMapping(value = "/add_project", method = RequestMethod.GET)
+    public String getAddProjectForm(Model model) {
+        model.addAttribute("project", new Project());
+        return "project/add_project";
     }
 
-    public boolean deleteProject(long id) {
-        return projectService.deleteProject(id);
+    @RequestMapping(value = "/add_project", method = RequestMethod.POST)
+    public String addProject(Project project,
+                             Errors errors,
+                             Principal principal) {
+        if (errors.hasErrors()) {
+            return "project/add_project";
+        }
+        // TODO: get authenticated user once user authorization is ready
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)